### PR TITLE
Create "Culture at CtC" page

### PR DESCRIPTION
**Pull request by nadintamer**
Creates a page that includes our core values and links to the page from the navigation bar.

### CODE REVIEW

File: src/views/Culture.vue [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+## Community
+
+- We respect, value, support, and cherish each other.
+- We welcome members of all backgrounds and abilities, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation (The Contributor Covenant). We have no tolerance for discrimination on the basis of these characteristics, and we accept the responsibility to prevent harassment, abuse, and intimidation of any kind within our organization.
+- We cultivate empathy and kindness for each other, our partners, and the broader community.
+- We are honest and forthright with each other, our partners, and all others with whom we interact. We strive to operate transparently wherever possible.
+
+## Humility
+
+- We forge reciprocal relationships with our partners. We help and learn from each other in a bilateral exchange, not a unilateral movement of aid from us to them.
```
Comment (U8NWXD) [inline src/views/Culture.vue:32]: I just noticed that in this line we use "we" and "each other" to refer both to us as a club and the union of us and our partners. Maybe clarify like this:
```suggestion
- We forge reciprocal relationships with our partners. We and our partners help and learn from each other in a bilateral exchange, not a unilateral movement of aid from us to them.
```

File: src/views/Culture.vue [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+- We are fallible, and in fact we will surely make mistakes. We create a space with safeguards and a culture that makes those mistakes okay.
+- We recognize that we need each other’s help, so we ask for help and encourage each other to do likewise.
+
+## Positive Impact
+
+- We commit to ensuring our work has a positive impact for people who need it.
+- We are mindful of the impact our work has, and we accept responsibility for that impact, both good and bad.
```
Comment (U8NWXD) [inline src/views/Culture.vue:39]: Maybe we should also emphasize that unintentional impact is part of this. For example:
```suggestion
- We are mindful of the impact our work has, and we accept responsibility for that impact, both good and bad, intentional and unintentional.
```

File: src/views/Culture.vue [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+- Our partners rely on us, and we rely on each other. Therefore, we accept the responsibility to deliver what we promise.
+- We hold ourselves accountable to each other, our partners, and our community for the impacts of our choices.
+- We clearly communicate our expectations and capabilities to each other and our partners.
+
+## Self-Care
+
+- We understand that we cannot effectively work with our partners unless we also care for each other and ourselves.
+- We make time to care for each other.
+- We recognize that we all need to practice self-care, so we encourage and support each other in practicing it.
```
Comment (U8NWXD) [inline src/views/Culture.vue:51]: I just realized that in this section we don't actually say that we take care of ourselves. Should we include it?

File: src/views/Culture.vue [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+# Addressing Failures and Improving our Culture
+
+We are always striving to improve our culture, and suggestions from you about how we can improve or areas where we are failing are important to us. Similarly, we want to hear about any concerns you have regarding actions or situations that contravene the spirit of these values. Please direct these suggestions and concerns to any of the following:
```
Comment (U8NWXD) [inline src/views/Culture.vue:55]:
```suggestion
We are always striving to improve our culture, and suggestions from you about how we can improve or areas where we are failing are important to us. Similarly, we want to hear about any concerns you have regarding actions or situations that contravene these values. Please direct these suggestions and concerns to any of the following:
```
no reason to limit ourselves to the spirit

File: src/App.vue [hunk 5]
```diff
@@ -7,6 +7,7 @@
       <div id="links">
         <span class="nav-links">
           <router-link class="link-text nav-text color-change" to="/people" @click.native="toggleRes(false)">people</router-link>
+          <router-link class="link-text nav-text color-change" to="/culture" @click.native="toggleRes(false)">culture</router-link>
```
[the remark this answers is not in the document]
Comment (nadintamer) [reply src/App.vue:10]: Done!

File: src/views/Culture.vue [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+- We learn from each other and our partners, and we enable others’ learning by sharing our skills, knowledge, and perspectives.
+- We support each other's growth by making space for each other, and embracing our own and each other’s mistakes as opportunities for growth.
+- We give constructive feedback kindly and receive it graciously.
+- We take time to reflect and improve.
+
+## Community
+
+- We respect, value, support, and cherish each other.
+- We welcome members of all backgrounds and abilities, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation (The Contributor Covenant). We have no tolerance for discrimination on the basis of these characteristics, and we accept the responsibility to prevent harassment, abuse, and intimidation of any kind within our organization.
```
[the remark this answers is not in the document]
Comment (nadintamer) [reply src/views/Culture.vue:26]: Not sure about this one? I agree that citing is important, but I feel like having quotation marks/even just having the place we cited it from in parentheses makes this seem a lot less sincere, so I'm conflicted. Any thoughts?

File: src/views/Culture.vue (hunk 4 above)
Comment (nadintamer) [reply src/views/Culture.vue:55]: Done!

File: src/views/Culture.vue (hunk 2 above)
Comment (nadintamer) [reply src/views/Culture.vue:39]: Yes, agreed. Added it!

File: src/views/Culture.vue (hunk 1 above)
Comment (nadintamer) [reply src/views/Culture.vue:32]: Done!

File: src/views/Culture.vue [hunk 7]
```diff
@@ -0,0 +1,110 @@
+<template>
+  <div class="container-fluid">
+    <div class="intro">
+      <h1 id="hook">Culture at CtC</h1>
+      <p class = 'hook-sub'>
+        Code the Change is about more than just technical work—we strive continuously to incorporate our core values in our club culture.
```
[the remark this answers is not in the document]
Comment (nadintamer) [reply src/views/Culture.vue:6]: Yeah, I like that. Done!

File: src/views/Culture.vue (hunk 3 above)
Comment (nadintamer) [reply src/views/Culture.vue:51]: I would say yes. We can include it in the second point: "We make time to care for each other and ourselves."

File: src/views/Culture.vue (hunk 6 above)
[the remark this answers is not in the document]
Comment (DrewGregory) [reply src/views/Culture.vue:26]: Agreed. I think a footnote saying that this line is inspired from the Contributor Covenant would be the best of both worlds there.

Apparently, footnotes aren't natively supported in Markdown, but you can use the Unicode character: https://stackoverflow.com/questions/25579868/how-to-add-footnotes-to-github-flavoured-markdown

[the remark this answers is not in the document]
Comment (U8NWXD) [reply src/views/Culture.vue:26]: In the Google Doc, we also had a number of other sources that I pulled from. What if we re-wrote this line so it's not a direct quote and then included a section at the end citing the places we pulled inspiration from, including the contributor covenant? That way we don't have to deal with footnotes

File: src/views/Culture.vue (hunk 3 above)
Comment (U8NWXD) [reply src/views/Culture.vue:51]: I like that!

Comment (nadintamer) [reply src/views/Culture.vue:51]: Done!

Comment (U8NWXD) [reply src/views/Culture.vue:51]: Can you check that this one is done? I'm not seeing the change in the diff

**Pull request merged**